webpack.config.js

```diff
@@ -49,7 +52,7 @@ const config = function(mode) {
         ignored: /node_modules/
       },
       contentBase: "public",
-      compress: true,
+      // compress: true,
```

Comment:
Just curious, why did you remove this line?

Reply:
These options are for devServer.
The production version will be automatically compressed.
I will change more on the webpack.config.js